sale_order_value_proposal_by_partner_shipping/models/sale_order.py

```diff
@@ -18,9 +18,3 @@ def _onchange_partner_shipping_id(self):
                 self.warehouse_id = partner.warehouse_id
             self.carrier_id = partner.property_delivery_carrier_id.filtered("active")
         return res
-
-    @api.onchange("partner_id")
-    def onchange_partner_id_carrier_id(self):
-        # if self.partner_id:
-        #     self.carrier_id = self.partner_id.property_delivery_carrier_id.filtered('active') # noqa
-        return
```

Comment:
Wasn't this disabling the standard logic? If so, this change must have been done on purpose.